README.md

```diff
@@ -85,6 +85,32 @@ Running the below command a 2nd time will overwrite the 1st wallet.
 
 Bear in mind that with this method, Wallet file is never written to host system.
 
+### Automatically load wallet entities
+
+#### On a detached setup
+
+We need a wallet with BOTH a balance and a pre-existent Public Drive. 
```

Comment:
It seems to me like too much ardrive-cli testing logic/data/setup/etc. is ending up in this repo. It seems like all of this should be in the cli repo and that this repo should be dedicated to setting up a clean execution environment to run the CLI and BATS tests. The testing scripts in the CLI repo should do all the test environment setup that it needs for the tests to run on any particular branch. It's getting too disjointed.